contrail-openstack/files/get_openstack_plugins_path.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+component=$1
+case $component in
+     neutron)      
+          plugins_path="neutron_plugin_contrail/extensions"
```

Comment:
we need to return where neutron_plugin_contrail of vnc_api is placed (like /usr/local/lib/python2.7) - without exact component. the rest of the path should be added in the caller.
but component should be passed to this file anyway to detect.
(just call 'import $component' and check result - make this func stupid)